packages/edit-site/src/components/keyboard-shortcuts/edit-mode.js

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
I didn't make any change to these three files, I just split the previous "index.js" file into three components in order to be able to insert them in different contexts. That way we can always make sure that the "save shortcut" works in all modes.